tools/pyinstaller/raiden_webapp.spec

```diff
@@ -66,3 +66,10 @@ exe = EXE(
     runtime_tmpdir=None,
     console=True,
 )
+
+if sys.platform == "darwin":
+    app = BUNDLE(
+        exe,
+        name="raiden_wizard.app",
```

Comment:
Mac apps usually have 'human readable' names (e.g. `Raiden Wizard.app` instead of `raiden_wizard.app`)

Reply:
Ah, you're right, I'll update the name.